### PR TITLE
Knative grpc

### PR DESCRIPTION
**Pull request by sriumcp**
I will remove the draft status from [my Knative PR](https://github.com/knative/docs/pull/4701) once this PR is merged and a 0.8 version is created after the merge.

### CODE REVIEW

File: mkdocs/docs/tutorials/load-test-grpc/community/knative/loadtest.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+???+ note "Before you begin"
+    1. [Complete the load test with SLO validation tutorial for Knative HTTP service](../../../load-test-http/community/knative/loadtest.md).
+    2. Update the Knative HTTP service used in the above tutorial to a Knative gRPC service as follows.
+    ```shell
+    kn service update hello \
+    --image docker.io/grpc/java-example-hostname:latest \
+    --port 50051 \
+    --revision-name=grpc
+    ```
```
Comment (kalantar) [inline mkdocs/docs/tutorials/load-test-grpc/community/knative/loadtest.md]: Is it necessary to run the HTTP test first? Would it be clearer if the instructions were:
1. Install Iter8
2. Install Knative and deploy your first Knative service
3. modify the service to be a gRPC service as follows:

**Review by kalantar — DISMISSED** [review]
Just a single comment. It is a suggestion. I'll approve so you can merge if you decide to ignore.

File: mkdocs/docs/tutorials/load-test-grpc/community/knative/loadtest.md (hunk 1 above)
Comment (sriumcp) [reply mkdocs/docs/tutorials/load-test-grpc/community/knative/loadtest.md]: Agreed. Let me fix... and push again.

**Review by kalantar — APPROVED** [review]
lgtm

**Pull request merged**